Tests/SentryProfilerTests/SentryProfilerTests.mm

```diff
@@ -153,9 +145,8 @@ - (void)testProfilerMutationDuringSlicing
     mutateExpectation.expectedFulfillmentCount = operations;
 
     void (^mutateBlock)(void) = ^(void) {
-        processBacktrace(backtrace, resolvedThreadMetadata, resolvedQueueMetadata, resolvedSamples,
-            resolvedStacks, resolvedFrames, frameIndexLookup, stackIndexLookup);
-        [mutateExpectation fulfill];
+        [state mutate:^(
+            __unused SentryProfilingMutableState *mutableState) { [mutateExpectation fulfill]; }];
```

Comment:
@indragiek I just noticed this change when going to write some new tests. Is the last backtrace created above ([here](https://github.com/getsentry/sentry-cocoa/pull/3035/files#diff-bfe2315bfeb5647454465dc52d0a0ba09dd139140e3292db7054a329a946e22bL149)) no longer appended to the profile data?